README.md

```diff
@@ -58,6 +59,17 @@ Make font-based queries useless for distinguishing any two users running:
 * on the same version of the same operating system
 * in the same [locale](#locale).
 
+This identifiability should be minimized as much as possible, with as little
+harm as possible to the following use cases:
+
+* Web users who have been installing web fonts over cheap connections to avoid
+  spending money or time downloading them on expensive or slow connections,
+  shouldn't now have to use the expensive or slow connections.
```

Comment:
Is this the right change, @tabatkins @pes10k?

```suggestion
* Web users who read minority languages who have been installing web fonts over cheap connections to avoid
  spending money or time downloading them on expensive or slow connections,
  shouldn't now have to use the expensive or slow connections.
```

Reply:
```suggestion
* Web users with cheap connections who use badly-supported minority languages,
  and thus have been installing fonts for their language locally
  to avoid spending money or time downloading them on expensive or slow connections,
  shouldn't now have to use the expensive or slow connections.
```

Reply:
Thanks!

Reply:
I would prefer merging this with the following item, buy

1) removing the existing first bullet
2) change the second bullet to read

Websites that depend on a font that's commonly installed among their visitors, but uncommon on the web, should have a way to continue working, _without those users spending money or time downloading those fonts on expensive or slow connections_

Reply:
It looks like this got merged into my PR in the middle of the conversation.  Can you kindly revert?

Reply:
These two bullets focus on different members of the priority of constituencies, so I think it works better to keep them separate. With Tab's edits, the first also focuses on users of minority languages, which the second doesn't, which is also a valuable distinction.

Sorry for merging prematurely. I'd thought the discussion between you and Tab had come to a conclusion.

Reply:
Thank you @jyasskin.  I dont mean to be pedantic, i'm just trying to make clear whether (and if so, why) perf costs are relevant in general, or just for lang-specific groups.

If the goals are (i) much better privacy preserving default, (ii) that doesn't harm folks who rely on non-OS fonts, (iii) allows expert users [dev or otherwise] to get perf / dev improvements through additional steps, wdyt of just reframing this whole section accordingly (im happy to suggest text if so)

Reply:
That's roughly how this section is framed now. The pre-existing first paragraph is (i), the Web Users bullet here is (ii) (ideally limited to minority-language users), and while I don't really care about (iii), you added it in https://github.com/w3cping/font-anti-fingerprinting/pull/6/commits/8520a58937c4dda1d3aff96d9522313193fd6cc5#diff-04c6e90faac2675aa89e2176d2eec7d8R64-R68, and I'm happy to either keep or drop it. Proposing new wording is, of course always helpful to see concretely what you think would be better.

Reply:
I would also be glad to drop (iii)!  I thought it was a CSSWG-side requirement.  @tabatkins is that wrong?  I (though i) remember it being discussed during that call.

But if its not needed, im more than happy to drop it too, and just rework the text accordingly

Reply:
No, I definitely never talked about expert users, and it's not something the CSSWG finds necessary either. Webfonts work just fine for that group.

Reply:
I am not certain that is the case. I see at least four people in this thread talking about finding a way to support downloading webfonts on cheaper connections: https://github.com/w3c/csswg-drafts/issues/4055

Reply:
Along with a good amount of pushback. ^_^ It's not currently the CSSWG consensus that this case is necessary to address, as I said.

Reply:
Right - there's no consensus either way.

Reply:
> I definitely never talked about expert users, and it's not something the CSSWG finds necessary either.

It is something that @r12a and @aphillips talked about and argued for, however